Doc 1457/add example mutation for activatephysicalcard to the doc (#521)
swan-io/docs: docs/topics/projects/index.mdx

The following tables list information about the available text message and email notifications.
Take note of the [account membership permissions](../accounts/memberships/index.mdx#permissions) required to receive each notification.
Account members won't receive the notification if they don't have the required permission.

> You can query your project's notification reminder planning settings (timing intervals for reminder notifications) using the `notificationReminderPlannings` query.

Author: ```suggestion
Use the `notificationReminderPlannings` query to check your project's reminder schedule.
```
